README.md

```diff
@@ -1 +1,77 @@
-# Pubsub
+# Pub/Sub to Logz.io
+
+Google Cloud Platform (GCP) Logging collects logs from your cloud services. You can use Google Cloud Pub/Sub to forward your logs from GP sinks to Logz.io.
+
+## Prerequisites
+
+-   Installed [gcloud CLI](https://cloud.google.com/sdk/docs/install) installed
+-   Active GCP account
+-   Installed [jq](https://stedolan.github.io/jq/download/)
+
+## Resources
+
+-   Cloud Build
+-   Pub/Sub
+-   Cloud Function
+-   Log Sink
+
+## Usage
+
+1. Log in to your GCP account:
```

Comment:
Please move step 1-3 to the prerequisites under - "Make sure you are connected to the relevant GCP project" and use html tag - details to expands the instruction on how to set it